platform/gold/GoldPlatform.js

```diff
@@ -33,8 +32,10 @@ class GoldPlatform extends Platform {
     return "gold";
   }
 
-  getLSBCodeName() {
-    return execSync("lsb_release -cs", {encoding: 'utf8'}).trim();
+  async getLSBCodeName() {
```

Comment:
`isUbuntu20()` still uses this in a sync manner

Reply:
also it's probably more natural to use try..catch instead of then..catch

Reply:
fixed isUbuntu20() and all downstream callers, but leave then..catch for now